src/v3/test/integration/__snapshots__/admin-consent-without-logo.test.tsx.snap

```diff
@@ -83,9 +86,9 @@ exports[`admin-consent-without-logo should render form without logo 1`] = `
                       class="MuiBox-root emotion-18"
                     >
                       <p
-                        class="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph emotion-19"
+                        class="MuiTypography-root MuiTypography-body1 emotion-12"
                         data-se="consent-description"
-                        id="admin-consent_Description_Allowing_access_will_share_1"
+                        tabindex="-1"
```

Comment:
So as part of the fix in Odyssey to allow the heading/`Title` component to be focusable, I added `tabindex="-1"` on their root Typography element. This had the side effect of making all uses of Odyssey `Typography` have `tabindex="-1"`, but AFAICT this doesn't cause any bad side effects for end users because the only thing that a negative tab index does is allow programmable focus to be set with JavaScript which is obviously only something that a downstream developer would make a conscious choice to do when using Odyssey `Typography`.

LMK if I'm overlooking something with that though!

Reply:
That makes sense, I think we are good with this then 🙏